qt/Application.cc

```diff
@@ -116,283 +151,290 @@ QAccessibleInterface* accessibleFactory(QString const& className, QObject* objec
 
 } // namespace
 
-Application::Application(int& argc, char** argv)
+Application::Application(int argc, char** argv, int& exit_code)
     : QApplication{ argc, argv }
-    , config_name_{ QStringLiteral("transmission") }
-    , display_name_{ QStringLiteral("transmission-qt") }
 {
-    setApplicationName(config_name_);
-    loadTranslations();
-    initUnits();
+    {
```

Comment:
What is this new nesting scope for?

It looks like it's to force local variables to go out-of-scope before `exec()` is called, but the only things affected by that seem to be a handful of `QString` and `QDir` variables that probably aren't affecting anything, so there's probably something else I'm not seeing

Reply:
If we do stick with this approach (see @mikedld's suggestion about `tr_main()` that would restructure this), maybe extract this all into a pre-exec function instead of nesting it like this?